extensions/api/data-management/asset/src/main/java/org/eclipse/dataspaceconnector/api/datamanagement/asset/model/DataAddress.java

```diff
@@ -0,0 +1,49 @@
+package org.eclipse.dataspaceconnector.api.datamanagement.asset.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+
+import java.util.Map;
+
+@JsonDeserialize(builder = DataAddress.Builder.class)
+public class DataAddress {
+
+    private Map<String, Object> properties;
+
+    private DataAddress(){
+    }
+
+    public Map<String, Object> getProperties() {
+        return properties;
+    }
+
+    public void setProperties(Map<String, Object> properties) {
+        this.properties = properties;
+    }
```

Comment:
I discussed this topic already with @paullatzelsperger here: https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/pull/641/files/fa66170632c2af2761d69bfc2499744d9125ee1f#r802525767
We don't need any public setter in the model classes
```suggestion
```